app/gui2/src/components/GraphEditor.vue

```diff
@@ -662,6 +662,8 @@ const groupColors = computed(() => {
     @click="handleClick"
     @dragover.prevent
     @drop.prevent="handleFileDrop($event)"
+    @pointermove.capture="graphNavigator.pointerEventsCapture.pointermove"
+    @wheel.capture="graphNavigator.pointerEventsCapture.wheel"
```

Comment:
This should also work

```suggestion
    v-on.capture="graphNavigator.pointerEventsCapture"
```

Reply:
That was what I tried originally, but `v-on.capture` doesn't seem to attach the bindings at all.